src/Goteo/Controller/Admin/ImpactDataAdminController.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/*
+ * This file is part of the Goteo Package.
+ *
+ * (c) Platoniq y Fundación Goteo <fundacion@goteo.org>
+ *
+ * For the full copyright and license information, please view the README.md
+ * and LICENSE files that was distributed with this source code.
+ */
+
+namespace Goteo\Controller\Admin;
+
+use Goteo\Application\Exception\ModelNotFoundException;
+use Goteo\Application\Message;
+use Goteo\Library\Text;
+use Goteo\Library\Forms\Model\ImpactDataForm;
+use Goteo\Model\ImpactData;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Route;
+
+class ImpactDataAdminController extends AbstractAdminController
+{
+    protected static $icon = '<i class="fa fa-2x fa-table"></i>';
+
+    public static function getGroup(): string {
+        return 'communications';
+    }
+
+    public static function getRoutes()
+    {
+        return [
+            new Route(
+                '/',
+                ['_controller' => __CLASS__ . "::listAction"]
+            ),
+            new Route(
+                '/edit/{id}',
+                ['_controller' => __CLASS__ . "::editAction"]
+            ),
+            new Route(
+                '/add',
+                ['_controller' => __CLASS__ . "::editAction"]
+            ),
+            new Route(
+                '/delete/{id}',
+                ['_controller' => __CLASS__ . "::deleteAction"]
+            )
+        ];
+    }
+
+    public function listAction(Request $request)
+    {
+        $page = $request->query->get('pag') ?: 0;
+        $limit = 10;
+        $list = ImpactData::getList([], $page * $limit, $limit, false);
+        $total = ImpactData::getList([], 0, 0, true);
+        
+        return $this->viewResponse('admin/impact_data/list',[
+            'list' => $list,
+            'total' => $total
+        ]);
+    }
+
+    public function editAction($id = '', Request $request)
```

Comment:
The $request param should be first here. Parameters with predefined values should always be at the end (because they are optional).